### DISCUSSION TITLE
Get Saver Variables Data from HA backup

### DISCUSSION THREAD

**Discussion by drfinn15** [PiotrMachowski/Home-Assistant-custom-components-Saver#22] [Q&A] [answered]
Hi,
I am having a problem with my HA installation as my Raspberry does not boot anymore. I know that my config and so on will get restored from the full backup I had scheduled every night, but I am not sure about my saver variables. As I saved a lot of data in my variables it would be crucial for me to have that data available, also after restoring the full backup. I therefore tried to pull the saver variables data from the db file in the backup .tar-file, but couldn't find any of the values. I saw some in the event_data table, but those variables with more data stored did not show up there either. Is there a way to get the saver variables content out of the db-file or any other part of the full backup?

**Comment by PiotrMachowski** [comment, marked as answer]
They should be in `state_attributes` table in DB. You can also try to check `/config/.storage/core.restore_state` file

**Reply by drfinn15** [reply]
Thank you so much! I couldn't find the content in the `state_attributes` table, but everything was stored in `/data/.storage/core.restore_state`.

**Reply by PiotrMachowski** [reply]
Great 👍
